mmtrack/datasets/mot_challenge_dataset.py

```diff
@@ -240,6 +245,85 @@ def format_bbox_results(self, results, infos, resfile):
                         f'{(y2-y1):.3f},{conf:.3f}\n')
             f.close()
 
+    def get_MOTChallenge_configs(self, tracker_folder, do_proc):
+        """Get default configs for trackeval.datasets.MotChallenge2DBox."""
```

Comment:
The comments are ambiguous and incomplete. [example](https://github.com/open-mmlab/mmtracking/blob/master/mmtrack/core/evaluation/eval_sot_vot.py#L17)